subjects/cat.en.md

```diff
@@ -18,13 +18,15 @@ Write a program that has the same behaviour as the system's `cat` command-line.
 
 `"Alan Mathison Turing was an English mathematician, computer scientist, logician, cryptanalyst. Turing was highly influential in the development of theoretical computer science, providing a formalisation of the concepts of algorithm and computation with the Turing machine, which can be considered a model of a general-purpose computer. Turing is widely considered to be the father of theoretical computer science and artificial intelligence."`
 
+-   In case of error the program should print the error.
+
 -   The program must be submitted inside a folder named `cat`.
 
 ```console
 student@ubuntu:~/piscine-go/cat$ go build
-student@ubuntu:~/piscine-go/cat$ ./cat Hello There
+student@ubuntu:~/piscine-go/cat$ ./cat
+Hello
```

Comment:
Not sure about this one but based on what I read in discord, shouldn't the two lines with "hello" wirtten there be removed from here??

Reply:
Now it works with the stdin and stdout, so you can run the command and write in it